script.js

```diff
@@ -0,0 +1,60 @@
+function toggleHiddenElements() {
+  const elements = [
+    '.menu-bars',
+    'main',
+    'footer',
+    'h1',
+  ];
+
+  elements.forEach((element) => {
+    document.querySelector(element).classList.toggle('hide');
+  });
+}
+
+function toggleShownElements() {
+  const elements = [
+    '#desktop-nav',
+    '#desktop-nav ul',
+  ];
+
+  elements.forEach((element) => {
+    document.querySelector(element).classList.toggle('show');
+  });
+}
+
+function removeCloseBtn() {
+  const closeBtn = document.querySelector('.close-btn');
+  closeBtn.parentNode.removeChild(closeBtn);
+}
+
+function closeMenu() {
+  document.querySelector('header').classList.remove('menu-visible');
+  toggleShownElements();
+  toggleHiddenElements();
+  removeCloseBtn();
+}
```

Comment:
- [ ] Kindly, make sure that your `X` can make the menu disappears when clicked as requested by the previous reviewer [here](https://github.com/Abenezer-Tilahun/Training-mobile-menu/pull/1/files/c655c63346f1fa8a0fb78fd4694a586b1dd67840#r942269655) It seems that we can not be able to close the mobile menu by clicking on the `X` button.